src/cryptonote/oaes_lib.c

```diff
@@ -490,7 +490,8 @@ static uint32_t oaes_get_seed(void)
   uint32_t _ret = 0;
 
   gettimeofday(&tv, NULL);
-  gmTimer = gmtime( &tv.tv_sec );
+  uint64_t tv_sec = tv.tv_sec;
+  gmTimer = gmtime( &tv_sec );
```

Comment:
Referencing &tv.tv_sec here was returning a null pointer and then line 497 below would crash.  This was due to a 32 vs 64 bit field in the structure depending on the compilation.